exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -96,6 +96,27 @@ fun <T : Any?> ExpressionWithColumnType<T>.varPop(scale: Int = 2): VarPop<T> = V
  */
 fun <T : Any?> ExpressionWithColumnType<T>.varSamp(scale: Int = 2): VarSamp<T> = VarSamp(this, scale)
 
+// using `Op`
+
+/** Returns this array of data wrapped in the `ALL` operator. */
+fun <T> Array<T>.allOp(): Op<T> = AllAnyOp("ALL", this)
+
+/** Returns this array of data wrapped in the `ANY` operator. The name is explicitly distinguished from [Array.any]. */
+fun <T> Array<T>.anyOp(): Op<T> = AllAnyOp("ANY", this)
+
+// using `CustomFunction`
+
+/** Returns this array of data wrapped in the `ALL` operator. */
+fun <T> Array<T>.allFunction(): CustomFunction<T> = AllFunction(this)
+
+/** Returns this array of data wrapped in the `ANY` operator. The name is explicitly distinguished from [Array.any]. */
+fun <T> Array<T>.anyFunction(): CustomFunction<T> = AnyFunction(this)
+
+/** Checks if this expression is equal to any element from [array].
+ * This is a more efficient alternative to [ISqlExpressionBuilder.inList] on PostgreSQL and H2. */
+infix fun <T> ExpressionWithColumnType<T>.eqAny(array: Array<T>): Op<Boolean> =
+    this eq array.anyOp() // TODO or `array.anyFunction()`
```

Comment:
This won't be necessary once the functions above are switched to take an argument instead of the more verbose extension function.